src/style.css

```diff
@@ -1,3 +1,73 @@
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto/Roboto-Medium.ttf);
+}
+
 body {
   margin: 0;
+
+  --color-blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+.logo-link {
+  display: flex;
+}
+
+.logo-image {
+  height: 40px;
+  width: 40px;
+}
+
+.list {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.element {
+  margin-left: 20px;
+}
+
+.element:first-child {
+  margin-left: 0;
+}
+
+.link {
+  text-decoration: none;
+  color: black;
+  font-family: Roboto, sans-serif;
+  height: 60px;
+  font-weight: 500;
+  display: flex;
+  font-size: 12px;
+  text-align: center;
+  align-items: center;
+}
+
+.link:hover {
+  color: var(--color-blue);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  background-color: var(--color-blue);
+  height: 4px;
+  width: 37px;
```

Comment:
Tutaj nie dawałbym statycznej wartosci na width jak 37px; bo działa to w tym przypadku kiedy słowo w menu jest krótkie tak jak "Apple" jednak w przypadku "Smartphones" bedziemy mieli krótka kreseczke do długiego wyrazu :/

Daj tutaj width:100% zeby zawsze kreska była długosci wyrazu pod którym sie znajduje. Zeby to zadziałało musisz jeszcze dac position: relative na klase do ktorej jest przypiety twój :after